_posts/2018-08-09-deploying-esxi-through-foreman.md

```diff
@@ -207,7 +211,6 @@ Copy the boot.cfg file from the mountpoint to tftproot and edit it to add a pref
 
   sed -e "s#/##g" -e "3s#^#prefix=../boot/ESXi-6.7.0-8169922\n#" -i boot-ESXi-6.7.0-8169922.cfg
 
-  sed -i "/^kernelopt=/c\kernelopt=ks=http://${foreman_server_ip}/unattended/provision" boot-ESXi-6.7.0-8169922.cfg # Replace with actual IP/FQDN of Foreman server
```

Comment:
No need to add kickstart to boot.cfg. There was no way to get the tokenized provision link here (without using Foreman-hooks!). Although this worked for our case...some people reported issues.